### PR TITLE
YLP-1033 create metrics when write documents in both dbs

### PR DESCRIPTION
**Pull request by Nas84**
- add promhetheus instrumentation as a middleware for http requests
- add custom metrics in store
- update changelog

### CODE REVIEW

File: data/storeDEPRECATED/mongo/mongo.go [hunk 1]
```diff
@@ -35,6 +37,22 @@ var (
 	}
 )
 
+var writeDataToReadStore = promauto.NewHistogramVec(prometheus.HistogramOpts{
```
Comment (ydrisR) [inline data/storeDEPRECATED/mongo/mongo.go]: can you rename this so we know what it is?
For example when it is called like this `writeDataToReadStore.WithLabelValues("cbg").Observe(float64(elapsed_time))` It looks like an instruction to write data to a DB...

something like `bucketDataWritesMetrics` ?

**Review by ydrisR — CHANGES_REQUESTED** [review]

**Comment by ydrisR** [comment]
Looks good, just a minor comment on metrics variables naming

File: data/storeDEPRECATED/mongo/mongo.go (hunk 1 above)
Comment (Nas84) [reply data/storeDEPRECATED/mongo/mongo.go]: sure will add a metrics suffix

**Review by ydrisR — APPROVED** [review]
LGTM

**Pull request merged**